src/app/compose/draftdesk.service.ts

```diff
@@ -58,7 +58,7 @@ export class DraftFormModel {
         const ret = new DraftFormModel();
         ret.from = fromAddress.email;
         ret.mid = draftId;
-        ret.to = to;
+        ret.to = to ? MailAddressInfo.parse(to) : null;
```

Comment:
If we declare it as an array, and the default value is `[]` maybe we should default to `[]` over `null` here too?

Reply:
Oops, yes seems saner